AppDelegate.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  AppDelegate.swift
+//  srushtika-tut-fixes
+//
+//  Created by Srushtika Neelakantam on 06/01/2020.
+//  Copyright © 2020 Srushtika Neelakantam. All rights reserved.
+//
+
+import UIKit
+import Ably
+import UserNotifications
+
+let authURL = ""
+let API_KEY = ""
+let myClientId = "abc"
+let ablyClientOptions = ARTClientOptions()
+
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate, ARTPushRegistererDelegate {
```

Comment:
I would ideally remove the empty message handlers (where the function just has boilerplate commentary) to reduce the noise for those looking at this file. I think most of the methods on `UIApplicationDelegate` are optional.